scripts/release/download-experimental-build-commands/print-summary.js

```diff
@@ -5,23 +5,20 @@
 const clear = require('clear');
 const {join, relative} = require('path');
 const theme = require('../theme');
-const {getCommitFromCurrentBuild} = require('../utils');
 
-module.exports = async () => {
+module.exports = async ({build}) => {
   const commandPath = relative(
     process.env.PWD,
     join(__dirname, '../download-experimental-build.js')
   );
 
   clear();
 
-  const commit = await getCommitFromCurrentBuild();
-
   const message = theme`
     {caution An experimental build has been downloaded!}
 
     You can download this build again by running:
-    {path   ${commandPath}} --commit={commit ${commit}}
+    {path   ${commandPath}} --build={build ${build}}
```

Comment:
No sense for e.g. Firefox reviewer to have to re-run the commit-to-build check when we already have the build ID.